quantum/process_keycode/process_caps_word.c

```diff
@@ -0,0 +1,147 @@
+// Copyright 2021-2022 Google LLC
+// SPDX-License-Identifier: Apache-2.0
+// Original source: https://getreuer.info/posts/keyboards/caps-word
+
+#include "process_caps_word.h"
+
+bool process_caps_word(uint16_t keycode, keyrecord_t* record) {
+    if (keycode == CAPSWRD) { // Pressing CAPSWRD toggles Caps Word.
+        if (record->event.pressed) {
+            caps_word_toggle();
+        }
+        return false;
+    }
+
+#ifndef NO_ACTION_ONESHOT
+    const uint8_t mods = get_mods() | get_oneshot_mods();
+#else
+    const uint8_t mods = get_mods();
+#endif // NO_ACTION_ONESHOT
+
+    if (!is_caps_word_on()) {
+        // The following optionally turns on Caps Word by holding left and
+        // right shifts or by double tapping left shift. This way Caps Word
+        // may be used without needing a dedicated key and also without
+        // needing combos or tap dance.
+#ifdef BOTH_SHIFTS_TURNS_ON_CAPS_WORD
+        // On many keyboards, the Command feature is enabled by default,
+        // which also uses left+right shift. It can be configured to use a
+        // different key combination by defining IS_COMMAND(). So we raise
+        // an error if Command is enabled but IS_COMMAND() is *not* defined.
+#    if defined(COMMAND_ENABLE) && !defined(IS_COMMAND)
+#        error "Caps Word and Command cannot both use Left Shift + Right Shift. Please see: https://docs.qmk.fm/#/feature_caps_word?id=command"
```

Comment:
we don't really want links in messages, IIRC. 

But how about: 
```suggestion
#        pragma message "Caps Word and Command should not be enabled at the same time do to conflicting usage of Left Shift + Right Shift. Please disable Command, or ensure that `IS_COMMAND` is not set to (get_mods() == MOD_MASK_SHIFT) (both shifts)"

    if (IS_COMMAND()) { return true; }
```

This will still compile, but generate and notice at compile time.  Useful for those that do use the command feature.  

And should force exit if the command key is triggered, too.

Reply:
Thanks, drashna! Great suggestions. I revised the error to a message and rewrote the logic like

 * If Command is enabled and `IS_COMMAND` is **not** defined: Then `IS_COMMAND` will use the default left+right shift key combination. We print a warning and never use left+right shift to activate Caps Word.

* If Command is enabled and `IS_COMMAND` is defined: We don't print a warning, but just in case test `IS_COMMAND()` at runtime and avoid activating Caps Word and Command at the same time.

* (And if Command is disabled: no problem.)

Reply:
(Meta comment: I think this Command conflict is now nicely addressed and so marked this thread resolved, but I'm not sure about the GitHub etiquette with such things. Feel free to unresolve.)